acceptance/testdata/restorer/container/cache/committed/io.buildpacks.lifecycle.cache.metadata

```diff
@@ -0,0 +1 @@
+{"buildpacks":[{"key":"cacher_buildpack","version":"cacher_v1","layers":{"cached-layer":{"sha":"sha256:29acd01e6fba0963e796f63eb3b517c2b2c6fb2ebcca5180f2bdda82c8b922d0","data":null,"build":false,"launch":false,"cache":true}}},{"key":"unused_buildpack","version":"v1","layers":{"cached-layer":{"sha":"sha256:4974a59c1c25cacec0d8e2623104618edb3d3e892a2a140dd35044bc1151e185","data":null,"build":false,"launch":false,"cache":true}}}]}
```

Comment:
Yeah, I have mixed feelings about that. Generating the tars on the fly would mean that we aren't testing that lifecycle can read a tar written by lifecycle _during an actual export_. Maybe that isn't an important part of the test and isn't worth keeping the binary data. I sort of worry that the creation of the tar will become most of this test and have divergent behavior from reality. I'm not opposed to making this suggested change though.

Reply:
Hmm... if we want to test operating on the result of an actual export I'd be inclined to just run a build with the `creator` in the test setup. I think having a sort of magic file makes it hard to understand what the test is validating and how.

Reply:
I could take a stab at generating some of the test data on the fly and see how it starts to look. It would get rid of the binary + the filename problem on windows.

Reply:
I took a path of doing the `tar` of the expected cache _inside_ the `Dockerfile`. This, I think, will be useful as we could introduce `Dockerfile.windows` in the future that tars it up and names the file the way windows containers expect. We could explore using a volume, but this seemed like a good compromise with less go code to generate tarballs in tmp directories and whatnot. Check it out and let me know what you think!

Reply:
Hmm... I definitely like this better than hard coding the tar files :)

Did you take a look at `layers.Factory.DirLayer`? That seems pretty easy to hook into and might give exactly what we want (and be cross platform too).

Reply:
I did briefly look at that helper. It could likely be leveraged, but not without some changes. The helper was building tars with paths like `/Users/myusername/some/path...`, which would not restore in the expected `/layers` location once inside the container and being restored by `restorer`.

Reply:
I see... in this case let's go with the Dockerfile approach, we can always modify or extend it later.